### PR TITLE
feature: lbg job submit, calc ef in vs test

### PR DESCRIPTION
**Pull request by dp-yuanyn**

### CODE REVIEW

File: scripts/submit_lbg_molecular_docking.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import os
+import shutil
+import json
+import subprocess as sp
+
+
+scass_type = "c16_m62_1 * NVIDIA T4"
+job_name = "test_molecular_docking_unidock1.0_t4"
+
+submit_dict = {
```
Comment (zhengh96) [inline scripts/submit_lbg_molecular_docking.py:19]: I think content with sensitive information should not be added to the repository, you can expose it as an input interface. @dp-yuanyn

File: scripts/submit_lbg_molecular_docking.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "project_id": 11053,
+    "dataset_path": ["/bohr/uni-dock-testdata-tn4t/v4"],
+    "command": "python test_molecular_docking.py --config_file config.json",
+    "out_files": ["results_dir/results.json", "results_dir/results.csv", "results_dir/metrics.csv"],
+    "platform": "ali",
+    "on_demand": 1,
+    "disk_size": 200,
+    "image_name": "registry.dp.tech/dp/vina_pipeline:0.1.10",
```
Comment (zhengh96) [inline scripts/submit_lbg_molecular_docking.py]: In addition, please try to use open-source and accessible images for the images, you can use the dptech dockerhub repository. @dp-yuanyn

**Review by caic99 — APPROVED** [review]
@dp-yuanyn LGTM. How about providing an example of submitting jobs?

File: scripts/submit_lbg_molecular_docking.py (hunk 1 above)
Comment (caic99) [reply scripts/submit_lbg_molecular_docking.py:19]: > I think content with sensitive information should not be added to the repository, you can expose it as an input interface. @dp-yuanyn

Username and password is configured in `lbg` ahead of running this script.

Comment (dp-yuanyn) [reply scripts/submit_lbg_molecular_docking.py:19]: @zhengh96 job name is set to input args now, and scass type, i think it is fine to keep in repo

File: scripts/submit_lbg_molecular_docking.py (hunk 2 above)
Comment (dp-yuanyn) [reply scripts/submit_lbg_molecular_docking.py]: > In addition, please try to use open-source and accessible images for the images, you can use the dptech dockerhub repository. @dp-yuanyn

changed

**Pull request merged**